pkgs/common-updater/scripts/update-source-version

```diff
@@ -165,7 +165,7 @@ else
     die "Couldn't figure out where out where to patch in new version in '$attr'!"
 fi
 
-if [[ "$oldHash" =~ ^(sha256|sha512)[:-] ]]; then
+if [[ "$oldHash" =~ ^(sha256|sha512|md5)[:-] ]]; then
```

Comment:
https://github.com/NixOS/nixpkgs/issues/77238

Reply:
Aha, fair point. I can't do anything about the md5 hash coming from the API, but I guess I can just make the update script ignore the updates for packages that only provide an md5 hash and force manual updates for these.

Reply:
Fixed. I realized that although having `update-source-version` download the file to hash does not work, I can do the same just fine in the grafana's update script myself.